app/actions/remote/user.ts

```diff
@@ -149,19 +145,19 @@ export const fetchProfilesPerChannels = async (serverUrl: string, channelIds: st
                     });
                 }
             }
-            modelPromises.push(operator.handleChannelMembership({
-                channelMemberships: memberships,
-                prepareRecordsOnly: true,
-            }));
-            const prepare = prepareUsers(operator, Array.from(users).filter((u) => u.id !== excludeUserId));
-            if (prepare) {
+            if (memberships.length) {
```

Comment:
Is this to avoid the warning? If not, I see no reason to check for the length (should be beautifully handled by the handle)

Reply:
Yes definitely to avoid the warning as it may very well be that membership is empty, same with users, as we may have already fetched the users for the channel, so they won't be loaded again.